driver/bpf/fillers.h

```diff
@@ -5778,6 +5786,107 @@ FILLER(sys_fchmod_x, true)
 	return bpf_val_to_ring(data, chmod_mode_to_scap(mode));
 }
 
+FILLER(sys_chown_x, true)
+{
+	/* Parameter 1: res (type: PT_ERRNO) */
+	long retval = bpf_syscall_get_retval(data->ctx);
+	int res = bpf_val_to_ring(data, retval);
+	CHECK_RES(res);
+
+	/* Parameter 2: path (type: PT_FSPATH) */
+	unsigned long path_pointer = bpf_syscall_get_argument(data, 0);
+	res = bpf_val_to_ring(data, path_pointer);
+	CHECK_RES(res);
+
+	/* Parameter 3: uid (type: PT_UINT32) */
+	u32 uid = (u32)bpf_syscall_get_argument(data, 1);
+	res = bpf_val_to_ring(data, uid);
+	CHECK_RES(res);
+
+	/* Parameter 4: gid (type: PT_UINT32) */
+	u32 gid = (u32)bpf_syscall_get_argument(data, 2);
+	return bpf_val_to_ring(data, gid);
+}
+
+FILLER(sys_lchown_x, true)
+{
+	/* Parameter 1: res (type: PT_ERRNO) */
+	long retval = bpf_syscall_get_retval(data->ctx);
+	int res = bpf_val_to_ring(data, retval);
+	CHECK_RES(res);
+
+	/* Parameter 2: path (type: PT_FSPATH) */
+	unsigned long path_pointer = bpf_syscall_get_argument(data, 0);
+	res = bpf_val_to_ring(data, path_pointer);
+	CHECK_RES(res);
+
+	/* Parameter 3: uid (type: PT_UINT32) */
+	u32 uid = (u32)bpf_syscall_get_argument(data, 1);
+	res = bpf_val_to_ring(data, uid);
+	CHECK_RES(res);
+
+	/* Parameter 4: gid (type: PT_UINT32) */
+	u32 gid = (u32)bpf_syscall_get_argument(data, 2);
+	return bpf_val_to_ring(data, gid);
+}
+
+FILLER(sys_fchown_x, true)
+{
+	/* Parameter 1: res (type: PT_ERRNO) */
+	long retval = bpf_syscall_get_retval(data->ctx);
+	int res = bpf_val_to_ring(data, retval);
+	CHECK_RES(res);
+
+	/* Parameter 2: fd (type: PT_FD) */
+	s32 fd = (s32)bpf_syscall_get_argument(data, 0);
+	res = bpf_val_to_ring(data, (s64)fd);
+	CHECK_RES(res);
+
+	/* Parameter 3: uid (type: PT_UINT32) */
+	u32 uid = (u32)bpf_syscall_get_argument(data, 1);
+	res = bpf_val_to_ring(data, uid);
+	CHECK_RES(res);
+
+	/* Parameter 4: gid (type: PT_UINT32) */
+	u32 gid = (u32)bpf_syscall_get_argument(data, 2);
+	return bpf_val_to_ring(data, gid);
+}
+
+FILLER(sys_fchownat_x, true)
+{
+	/* Parameter 1: res (type: PT_ERRNO) */
+	long retval = bpf_syscall_get_retval(data->ctx);
+	int res = bpf_val_to_ring(data, retval);
+	CHECK_RES(res);
+
+	/* Parameter 2: dirfd (type: PT_FD) */
+	s32 dirfd = (s32)bpf_syscall_get_argument(data, 0);
+	if(dirfd == AT_FDCWD)
+	{
+		dirfd = PPM_AT_FDCWD;
+	}
+	res = bpf_val_to_ring(data, (s64)dirfd);
+	CHECK_RES(res);
+
+	/* Parameter 3: pathname (type: PT_FSRELPATH) */
+	unsigned long path_pointer = bpf_syscall_get_argument(data, 1);
+	res = bpf_val_to_ring(data, path_pointer);
+	CHECK_RES(res);
+
+	/* Parameter 3: uid (type: PT_UINT32) */
+	u32 uid = (u32)bpf_syscall_get_argument(data, 2);
+	res = bpf_val_to_ring(data, uid);
+	CHECK_RES(res);
+
+	/* Parameter 4: gid (type: PT_UINT32) */
+	u32 gid = (u32)bpf_syscall_get_argument(data, 3);
```

Comment:
here you miss a `res = bpf_val_to_ring(data, gid);` this is the reason why tests are failing on arm64

Reply:
Thanks!